shared-ts/utils/dynamo.ts

```diff
@@ -95,3 +101,55 @@ export const getOrganisationsInfo = async (): Promise<Organisations | null> => {
         throw e;
     }
 };
+
+export const getAllOrganisationsInfo = async (): Promise<OrganisationsWithStats | null> => {
+    logger.info(`Getting all organisations from DynamoDB table...`);
+    try {
+        const dbDataInfo = await ddbDocClient.send(
+            new ScanCommand({
+                TableName: organisationsTableName,
+                FilterExpression: "SK = :info",
+                ExpressionAttributeValues: {
+                    ":info": "INFO",
+                },
+            }),
+        );
+
+        const dbDataStat = await ddbDocClient.send(
+            new ScanCommand({
+                TableName: organisationsTableName,
+                FilterExpression: "SK = :stat",
+                ExpressionAttributeValues: {
+                    ":stat": "STAT",
+                },
+            }),
+        );
+
+        const parsedOrgInfo = organisationsSchema.safeParse(dbDataInfo.Items);
```

Comment:
Rather than doing 2 parses and a map here, could we not get the data into the right format to use the `organisationsSchemaWithStats` schema? That way we only need a single parse and we shouldn't need the cast on line 143

Reply:
done